ts/private/ts_proto_library.bzl

```diff
@@ -123,7 +124,8 @@ ts_proto_library = rule(
             cfg = "exec",
         ),
         "protoc_gen_connect_es": attr.label(
-            doc = "protoc plugin to generate services",
+            doc = """protoc plugin to generate services
+            deprecated: no longer used now that @bufbuild/protoc-gen-es generates service definitions""",
```

Comment:
Might want to include the version number in the deprecation warnings. 

deprecated: no longer used bufbuild v2. Prior to v2 this plugin generated service definitions.